### PR TITLE
[SMALLFIX] Add runUfsTests docs and support local ufs

### PR DESCRIPTION
**Pull request by LuQQiu**
Add the docs for CLI runUfsTests and support running tests against local ufs.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@calvinjia Please take a look, thanks

**Comment by LuQQiu** [comment]
@madanadit Added the docs an tested against local ufs. PTAL, thanks

File: docs/en/basic/Command-Line-Interface.md [hunk 1]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
 ./bin/alluxio runTests
 ```
 
+### runUfsTests
+
+The `runUfsTests` aims to test the integration between Alluxio and the given ufs. The ufs tests
+contain all the Alluxio semantics with the ufs.
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: contain -> validate the semantics Alluxio expects of the UFS

File: docs/en/basic/Command-Line-Interface.md [hunk 2]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
...
+
+`--help` will provide the detail guidance on how to run this command.
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: provide detailed guidance

File: docs/en/basic/Command-Line-Interface.md [hunk 3]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
...
+`--path <ufs_path>` is required to provide the full ufs path to run ufs tests against.
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: `--path <ufs_path>` (required) the full UFS path to run tests against.

File: docs/en/basic/Command-Line-Interface.md [hunk 4]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
 ./bin/alluxio runTests
 ```
 
+### runUfsTests
+
+The `runUfsTests` aims to test the integration between Alluxio and the given ufs. The ufs tests
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: capitalize UFS throughout this doc

File: docs/en/basic/Command-Line-Interface.md [hunk 5]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
...
+contain all the Alluxio semantics with the ufs.
+
+`--help` will provide the detail guidance on how to run this command.
+`--path <ufs_path>` is required to provide the full ufs path to run ufs tests against.
+
+The usage of this command includes:
+* Test if the given ufs credentials are valid before having a running Alluxio cluster and mounting the ufs to Alluxio.
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: Test if the given UFS credentials are valid before mounting the UFS to an Alluxio cluster

File: docs/en/basic/Command-Line-Interface.md [hunk 6]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
...
+* If the given Ufs is S3, this test can also be used as a S3 compatibility test to test if the target under filesystem can
+  fulfill the minimum S3 compatibility requirements in order to work well with Alluxio through Alluxio's integration with S3.
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: remove "through Alluxio's integration with S3"

File: docs/en/basic/Command-Line-Interface.md [hunk 7]
```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
...
+* Test the integration between Alluxio and the given ufs is not broken. This usage is more related to Alluxio developers.
```
Comment (madanadit) [inline docs/en/basic/Command-Line-Interface.md]: I would reword this entire bullet "Validate the contract between Alluxio and the given UFS. This is primarily intended for Alluxio developers. Developers are required to add test coverage for changes to an Alluxio UFS module and run those tests to validate."

File: docs/en/basic/Command-Line-Interface.md (hunk 2 above)
Comment (LuQQiu) [reply docs/en/basic/Command-Line-Interface.md]: Changed, thanks

File: docs/en/basic/Command-Line-Interface.md (hunk 3 above)
Comment (LuQQiu) [reply docs/en/basic/Command-Line-Interface.md]: Changed, thanks

File: docs/en/basic/Command-Line-Interface.md (hunk 4 above)
Comment (LuQQiu) [reply docs/en/basic/Command-Line-Interface.md]: Capitalized all UFS.

File: docs/en/basic/Command-Line-Interface.md (hunk 5 above)
Comment (LuQQiu) [reply docs/en/basic/Command-Line-Interface.md]: Changed, thanks

File: docs/en/basic/Command-Line-Interface.md (hunk 6 above)
Comment (LuQQiu) [reply docs/en/basic/Command-Line-Interface.md]: Removed, thanks

File: docs/en/basic/Command-Line-Interface.md (hunk 7 above)
Comment (LuQQiu) [reply docs/en/basic/Command-Line-Interface.md]: Changed the entire bullet. Thanks! The docs are much easier to understand.

**Comment by LuQQiu** [comment]
Thanks Adit for improving the docs sentences!

@calvinjia do we want to cherry pick this PR to branch-2.0. It fix runUfsTests to support local ufs and added the runUfsTests docs.

**Comment by madanadit** [comment]
yes, we want to cherry-pick after PR build succeeds

**Comment by LuQQiu** [comment]
Jenkins, test this please

**Comment by madanadit** [comment]
can address @calvinjia 's comments post-merge

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.
